App/SB/views/ThreadsDetail/index.js

```diff
@@ -272,7 +272,7 @@ const mapDispatchToProps = (dispatch) => {
     dismissPhoto: () => { dispatch(UIActions.dismissViewedPhoto()) },
     viewPhoto: (photoId, threadId) => { dispatch(UIActions.viewPhotoRequest(photoId, threadId)) },
     showImagePicker: (threadId) => { dispatch(UIActions.showImagePicker(threadId)) },
-    refreshMessages: (hidden) => { dispatch(TextileNodeActions.refreshMessagesRequest(hidden)) },
```

Comment:
yeah, drop it. wild, there is def a clean up commit not included here. the idea was to avoid showing the spinner wheel on refresh during screen load. but kept it simple to start by just removing it